sympy/physics/vector/frame.py

```diff
@@ -621,13 +621,16 @@ def _rot(axis, angle):
                 from sympy.physics.vector.functions import kinematic_equations
                 q1, q2, q3 = amounts
                 u1, u2, u3 = symbols('u1, u2, u3', cls=Dummy)
```

Comment:
Yes. I meant put them in the `else` part of the block where they are used. In the `if` part they are being set to 0; they are used in the `else` part as you note.